### ISSUE TITLE
Bundle release broken: 'push bundles' step fails with exit code 254

### ISSUE THREAD

**Issue by bicknellr** [lit/lit#2545]
The bundles step of the latest release failed with a no-so-helpful error:
https://github.com/lit/lit/runs/5208044575?check_suite_focus=true#step:10:168

**Comment by bicknellr** [comment]
There's nothing particularly interesting in the diff with the last commit on which the release workflow was run that successfully published bundles (ff9d64e). Specifically, the only step that failed is the 'push bundles' step and the only changes there are to (a) the step id, which is working correctly because you can see the places where `publishedPackages` was interpolated correctly, (b) the bundle copying lines, which I have no reason to suspect, and (c) the removal of the `git config ...` lines, which shouldn't have any effect as these options are now set in an earlier step that is itself known to be working because the 'Version Packages' PR that this release was run from was committed as `lit-robot`.

```diff
diff --git a/.github/workflows/release.yaml b/.github/workflows/release.yaml
index 8e533ca0..bfb47fa4 100644
--- a/.github/workflows/release.yaml
+++ b/.github/workflows/release.yaml
@@ -7,36 +7,59 @@ on:
 
 jobs:
   release:
-    # Prevents changesets action from creating a PR on forks
+    # Don't run on forks. We can and should only release from the main repo.
     if: github.repository == 'lit/lit'
+
     name: Release
     environment: Changesets
     runs-on: ubuntu-latest
     steps:
-      - name: Checkout Repo
-        uses: actions/checkout@v2
-        with:
-          # This makes Actions fetch all Git history so that Changesets can generate changelogs with the correct commits
-          fetch-depth: 0
+      - name: Use Lit Robot's name and email for all git actions
+        run: |
+          git config --global user.name "Lit Robot"
+          git config --global user.email "lit-robot@google.com"
 
-      - name: Setup Node.js 14.x
+      - name: Setup Node.js 16
         uses: actions/setup-node@v2
         with:
-          node-version: 14.x
+          node-version: 16
+
+      # changesets/action@v1 currently has a bug where it attempts to set the
+      # local git user info before changing directories for `cwd`, which means
+      # it will fail if the root directory is not a git repo.
+      - name: Initialize empty root repo
+        run: git init
+
+      - name: Checkout Lit Repo
+        uses: actions/checkout@v2
+        with:
+          path: lit
+          # This makes Actions fetch all Git history so that Changesets can
+          # generate changelogs with the correct commits
+          fetch-depth: 0
 
       - name: Install Dependencies
+        working-directory: lit
         run: npm ci
 
       # Changeset uses prettier@1 which creates unstable formatting issues
       # Deleting this forces fallback to our version of prettier
       # https://github.com/changesets/changesets/issues/616
       - name: Remove old version of prettier from changesets
+        working-directory: lit
         run: rm -rf node_modules/@changesets/*/node_modules/prettier
 
       - name: Create Release Pull Request or Publish to npm
-        id: changesets
-        uses: changesets/action@master
+        # The id of this step must not be "changesets", or else the step will be invisible
+        # in the list of steps from the GitHub UI when the action runs (though it will still
+        # run, and its output will appear in the raw logs). Unknown why this is the case,
+        # see https://github.com/changesets/action/issues/149 for discussion.
+        id: cs
+        uses: changesets/action@v1
         with:
+          # The `working-directory` option is only available for steps that use
+          # `run`. This `cwd` option is the same, but specific to this action.
+          cwd: lit
           version: npm run version
           publish: npm run release
         env:
@@ -46,7 +69,9 @@ jobs:
 
       - name: Checkout dist repo
         uses: actions/checkout@v2
-        if: steps.changesets.outputs.published
+        # GitHub Actions outputs are strings and the string 'false' converts to
+        # `true` because it isn't the empty string.
+        if: steps.cs.outputs.published == 'true'
         with:
           repository: lit/dist
           ref: empty
@@ -54,12 +79,14 @@ jobs:
           token: ${{ secrets.LIT_ROBOT_AUTOMATION_PAT }}
 
       - name: Push bundles to dist repo
-        if: steps.changesets.outputs.published
+        # GitHub Actions outputs are strings and the string 'false' converts to
+        # `true` because it isn't the empty string.
+        if: steps.cs.outputs.published == 'true'
         working-directory: dist
         run: |
           # Extract the version of `lit` that was published or the empty string.
           LIT_VERSION=$(npm run --silent extract-published-lit-version <<EOF
-            ${{ steps.changesets.outputs.publishedPackages }}
+            ${{ steps.cs.outputs.publishedPackages }}
           EOF
           )
           # Don't create a bundle commit if `lit` wasn't published.
@@ -70,15 +97,13 @@ jobs:
           git checkout --detach empty
           # Copy in all of the bundles.
           mkdir all
-          cp ../packages/lit/lit.all.min.js all/lit-all.min.js
-          cp ../packages/lit/lit.all.min.js.map all/lit-all.min.js
+          cp ../lit/packages/lit/lit-all.min.js all
+          cp ../lit/packages/lit/lit-all.min.js.map all
           mkdir core
-          cp ../packages/lit/lit.min.js core/lit-core.min.js
-          cp ../packages/lit/lit.min.js.map core/lit-core.min.js
+          cp ../lit/packages/lit/lit-core.min.js core
+          cp ../lit/packages/lit/lit-core.min.js.map core
           # Stage the bundles, create the commit, tag it, and push.
           git add .
-          git config user.name "Lit Robot"
-          git config user.email "lit-robot@google.com"
           git commit -m "Bundles for lit@${LIT_VERSION}"
           git tag "v${LIT_VERSION}"
           git push origin "v${LIT_VERSION}"
```

**Comment by bicknellr** [comment]
Fixed by #2548.
